examples/readme.md

```diff
@@ -11,15 +11,15 @@ git clone https://github.com/pagantis/orders-api-client.git .
 composer install
 ```
 
-Then you can run the web application using PHP's built-in server:
+Then you can run the web application using PHP's built-in server via Composer Scripts:
 
 ```
-php -S 0.0.0.0:8000 -t examples/
+composer run-script examples-server
```

Comment:
After execute this script I receive this warning:
You made a reference to a non-existent script @putenv COMPOSER_PROCESS_TIMEOUT=0
You made a reference to a non-existent script @putenv PHP_CLI_SERVER_WORKERS=2

Reply:
removed fluff